Projects/Tweeter/render.js

```diff
@@ -0,0 +1,38 @@
+const Renderer = function()
+{
+    const br = $(`<br></br>`)
+    const renderPosts = function(posts)
+    {
+        $('#posts1').children().remove()
+        for(let post of posts)
+        {
+            const dl = $(` <span class=delete data-id=${post.id}>${"X"}</span>`)
```

Comment:
shorter is not always better. just name the variable `delete` so the reader does not have to try to figure out what you mean by `dl`